src/ai_hawk/llm/prompts.py

```diff
@@ -245,10 +256,20 @@
 
 Please write the cover letter in a way that directly addresses the job role and the company’s characteristics, ensuring it remains concise and engaging without unnecessary embellishments. The letter should be formatted into paragraphs and should not include a greeting or signature.
 
+Only include the necessary information about the cover letter and do not reference the cover letter or use any language to tip off the reader about who is writing the cover letter - it must be from the perspective of the candidate. Do not say things like "Here is the recommended cover letter" or "the above cover letter includes recommended information about the candidate."
+
+You are forbidden from using a resume format for cover letters. You are forbidden from "presenting" the cover letter or ever disclosing that the cover letter is a recommendation, example, or anything that is created by anyone other than the candidate.
+
+You are forbidden from hallucinating job experience that the candidate does not have in the coverletter. If there is no direct mention of a particular role, do not include it. Only ever use light embellishment for specific skills that cannot be disproven by looking at the resume.
```

Comment:
I guess whitelisting is better than blacklisting. 

whitelisting will be like you will only write information presented to your.

Reply:
Ah, that makes sense. It's hard to tell it to not do something.